packages/matrix-resolve/src/index.test.ts

```
@@ -1,153 +1,140 @@
-import fetch from 'node-fetch'
-import { resolve as dnsResolve, lookup } from 'node:dns'
-import { matrixResolve, MatrixResolve } from './index'
-import { type WellKnownMatrixServer } from './index'
+import fetch from "node-fetch";
+import { MatrixResolve, matrixResolve, type WellKnownMatrixServer } from "./index";
 
 type DnsResolve = (
```

Biome (2.4.9)

[error] This type alias DnsResolve is unused.

(lint/correctness/noUnusedVariables)